test/components/daily/BolusTooltip.test.js

```diff
@@ -627,12 +633,18 @@ describe('BolusTooltip', () => {
     expect(wrapper.find(formatClassesAsSelector(styles.target))).to.have.length(1);
   });
 
-  it('should render automated header label for automated bolus', () => {
+  it('should render an automated header label for automated bolus', () => {
     const wrapper = mount(<BolusTooltip {...props} bolus={automated} />);
     expect(wrapper.find(formatClassesAsSelector(styles.title))).to.have.length(1);
     expect(wrapper.find(formatClassesAsSelector(styles.title)).text()).to.include('Automated');
   });
 
+  it('should render a one-button bikys header label for a one-button bolus', () => {
```

Comment:
`bolus` but with all the right-hand letters shifted one to the left :)
```suggestion
  it('should render a one-button bolus header label for a one-button bolus', () => {
```

Reply:
Ha :)